### PR TITLE
Highlight invite_members_button until user interacts with it

### PR DESCRIPTION
**Pull request by neallred**
#### Summary
Invite members button is highlighted until the user interacts with it.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-38803 , but first intended to be included with
https://mattermost.atlassian.net/browse/MM-38258

#### Screenshots
invite-members-highlighted

#### Release Note
No release note; this has not been released and was intended to be part of the invite members button when we established that as the A/B test winner.

```release-note
NONE
```

### CODE REVIEW

File: components/sidebar/invite_members_button.test.tsx [hunk 1]
```diff
@@ -49,14 +49,19 @@ describe('components/sidebar/invite_members_button', () => {
         },
     };
 
+    const props = {
+        onClick: jest.fn(),
+        touchedInviteMembersButton: false,
```
Comment (marianunez) [inline components/sidebar/invite_members_button.test.tsx:54]: Could we add a test here for when it is touched? `touchedInviteMembersButton: true`

File: components/sidebar/sidebar_category/sidebar_category.test.tsx [hunk 2]
```diff
@@ -31,9 +31,12 @@ describe('components/sidebar/sidebar_category', () => {
         isNewCategory: false,
         isDisabled: false,
         limitVisibleDMsGMs: 10000,
+        touchedInviteMembersButton: false,
```
Comment (marianunez) [inline components/sidebar/sidebar_category/sidebar_category.test.tsx:34]: Same here, could we add a test case for `touchedInviteMembersButton = true`

**Review by marianunez — APPROVED** [review]
Great work @nathanaelhoun! 🎉 LGTM, just added comments to add new case in tests.

**Review by jasonblais — APPROVED** [review]
Tried with a System Admin account, LGTM :+1:

**Review by pablovelezvidal — APPROVED** [review]
Nice work @neallred ! :shipit:

**Review by stevemudie — APPROVED** [review]
- Created a new user and logged in
 - Noticed the highlighted invite link and clicked on it
 - Exited the invite modal and verified that the link was no longer highlighted as expected

Nice work @neallred!

**Comment by neallred** [comment]
Manually cherry-picked to [cloud](https://github.com/mattermost/mattermost-webapp/commits/cloud) and [release-6.0](https://github.com/mattermost/mattermost-webapp/commits/release-6.0)

**Pull request merged**